### PR TITLE
Display only active options in employee mapping page

### PR DESCRIPTION
**Pull request by anishfyle**

### CODE REVIEW

File: tests/test_fyle/test_views.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 from apps.workspaces.models import FyleCredential, Workspace
 import pytest
 import json
+from fyle_accounting_mappings.models import DestinationAttribute
```
Comment (ashwin1111) [inline tests/test_fyle/test_views.py:5]: remove this unused import

File: apps/quickbooks_online/views.py [hunk 2]
```diff
@@ -74,7 +74,7 @@ class EmployeeView(generics.ListCreateAPIView):
 
     def get_queryset(self):
         return DestinationAttribute.objects.filter(
-            attribute_type='EMPLOYEE', workspace_id=self.kwargs['workspace_id']).order_by('value')
+            attribute_type='EMPLOYEE',active=True, workspace_id=self.kwargs['workspace_id']).order_by('value')
```
Comment (ashwin1111) [inline apps/quickbooks_online/views.py:77]: indentation, once space after comma

File: apps/fyle/views.py [hunk 3]
```diff
@@ -267,7 +267,7 @@ class EmployeeView(generics.ListCreateAPIView):
 
     def get_queryset(self):
         return ExpenseAttribute.objects.filter(
-            attribute_type='EMPLOYEE', workspace_id=self.kwargs['workspace_id']).order_by('value')
+            attribute_type='EMPLOYEE',active=True, workspace_id=self.kwargs['workspace_id']).order_by('value')
```
Comment (ashwin1111) [inline apps/fyle/views.py:270]: indent, check below comment

**Review by ashwin1111 — APPROVED** [review]

File: apps/quickbooks_online/views.py (hunk 2 above)
Comment (anishfyle) [reply apps/quickbooks_online/views.py:77]: Yes, Making Changes.

File: apps/fyle/views.py (hunk 3 above)
Comment (anishfyle) [reply apps/fyle/views.py]: Done.

**Pull request merged**